modules/EvseManager/evse_energy_control/evse_manager_energy_controlImpl.hpp

```diff
@@ -1,5 +1,3 @@
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
How did the license header disappear here? Could this be a bug in ev-cli?

Reply:
solved by running edm mod update --force